src/apolo_app_types/protocols/bundles/llm.py

```diff
@@ -46,7 +48,7 @@ class LLMBundleInputs(AppInputs, typing.Generic[TSize]):
     This class can be extended by specific LLM bundle input classes.
     """
 
-    hf_token: ApoloSecret = Field(  # noqa: N815
+    hf_token: HuggingFaceToken = Field(  # noqa: N815
         ...,
         json_schema_extra=HF_TOKEN_SCHEMA_EXTRA.as_json_schema_extra(),
     )
```

Comment:
maybe, let's bake HuggingFaceToken and it's schema_extra together? for simplicity